resources/Components/KeyboardShortcutConfigurator/index.js

```diff
@@ -1,28 +1,22 @@
-import React, { useState, useEffect } from 'react'
+import React, { useEffect } from 'react'
 import hotkeys from 'hotkeys-js'
 
 import link from '../../../resources/link'
 import { getShortcutFromKeyEvent, getDisplayShortcut, isShortcutKey } from '../../../resources/app'
 
 const KeyboardShortcutConfigurator = ({ actionText = '', platform, shortcut, shortcutName }) => {
-  const [configuring, setConfiguring] = useState(false)
+  const { modifierKeys, shortcutKey } = getDisplayShortcut(platform, shortcut)
 
-  useEffect(() => {
-    hotkeys.unbind()
-    if (configuring) {
-      // set configuring so existing shortcut keypresses can be temporarily ignored
-      link.send('tray:action', 'setShortcut', shortcutName, {
-        ...shortcut,
-        configuring: true
-      })
+  const EnterShortcut = () => {
+    useEffect(() => {
+      hotkeys.unbind()
```

Comment:
Doesn't work in the callback

Reply:
how do you know its still bound? I tried it locally and seems to work as expected

Reply:
It's still listening for a shortcut after you've set one - so if you set `Alt + J` and then immediately press `Alt + K` without clicking to configure the shortcut, it will update the shortcut to `Alt + K`

Reply:
I'm not seeing that behavior. maybe the return was in the wrong place? gonna push up what I have and you can take it for a spin

Reply:
Looks exactly like what I had, will give it a go

Reply:
Ah yeah return was in the wrong place